tests/integration/annotation_import/conftest.py

```diff
@@ -152,14 +159,42 @@ def configured_project_pdf(client, ontology, rand_gen, pdf_url):
             where=LabelingFrontend.name == "editor"))[0]
     project.setup(editor, ontology)
     data_row_ids = []
-    data_row_ids.append(dataset.create_data_row(row_data=pdf_url).uid)
+    data_row_ids.append(dataset.create_data_row(pdf_url).uid)
     project.datasets.connect(dataset)
     project.data_row_ids = data_row_ids
     yield project
     project.delete()
     dataset.delete()
 
 
+@pytest.fixture
+def dataset_pdf_entity(client, rand_gen, pdf_entity_row_data):
+    dataset = client.create_dataset(name=rand_gen(str))
+    data_row_ids = []
+    data_row = dataset.create_data_row(pdf_entity_row_data)
+    data_row_ids.append(data_row.uid)
+    yield dataset, data_row_ids
+    dataset.delete()
+
+
+@pytest.fixture
+def configured_project_pdf_entity(client, ontology, rand_gen,
+                                  dataset_pdf_entity):
+    project = client.create_project(name=rand_gen(str),
```

Comment:
Couldn't we reuse `configured_project_without_data_rows` fixture here? Also, we should prefer `QueueMode.Batch` over `QueueMode.Dataset`, `QueueMode.Dataset` is an older paradigm.

Reply:
will look into both

Reply:
this is done